diff --git a/index.cjs b/index.cjs
index 9fc1f5cb6033dff429171980de81a9104f0f5024..9e43b1afa5e147ea75e5e5d251259cd9b3c557aa 100644
--- a/index.cjs
+++ b/index.cjs
@@ -1,7 +1,8 @@
-'use strict'
+"use strict"

-if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./dist/pinia.prod.cjs')
+console.warn("pinia patch test")
+if (process.env.NODE_ENV === "production") {
+  module.exports = require("./dist/pinia.prod.cjs")
 } else {
-  module.exports = require('./dist/pinia.cjs')
+  module.exports = require("./dist/pinia.cjs")
 }
diff --git a/index.js b/index.js
index 9fc1f5cb6033dff429171980de81a9104f0f5024..9e43b1afa5e147ea75e5e5d251259cd9b3c557aa 100644
--- a/index.js
+++ b/index.js
@@ -1,7 +1,8 @@
-'use strict'
+"use strict"

-if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./dist/pinia.prod.cjs')
+console.warn("pinia patch test")
+if (process.env.NODE_ENV === "production") {
+  module.exports = require("./dist/pinia.prod.cjs")
 } else {
-  module.exports = require('./dist/pinia.cjs')
+  module.exports = require("./dist/pinia.cjs")
 }
